packages/frontend/amp/components/Standfirst.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { headline } from '@guardian/pasteup/typography';
+import { css, cx } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { pillarPalette, pillarMap } from '@frontend/lib/pillars';
+
+// TODO - unclear if we need the list styles as well here
+const listStyles = (pillar: Pillar) => css`
```

Comment:
The css in this file is just copy-pasted from before, but I've never felt comfortable with this kind of styling (which in theory should be broader than standfirst) going on here. It's a general issue we should think about when we do the AMP tech review - how to share and express common text styling (beyond simply what's in the typography file at the moment - i.e. things like p tags and bullets/lists).

Reply:
I agree, it probably belongs in Pasteup.